### PR TITLE
[IMP] account: update urls into nice urls

### PR DESCRIPTION
**Pull request by sauc-odoo**
This commit modifies existing URLs to the new format 
so that the URL's become more human readable. 

Task-3820230

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/views/account_portal_templates.xml [hunk 1]
```diff
@@ -104,7 +104,7 @@
         <xpath expr="//div[hasclass('o_portal_sidebar')]" position="inside">
             <t t-set="o_portal_fullwidth_alert" groups="sales_team.group_sale_salesman,account.group_account_invoice,account.group_account_readonly">
                 <t t-call="portal.portal_back_in_edit_mode">
-                    <t t-set="backend_url" t-value="'/web#model=%s&amp;id=%s&amp;action=%s&amp;view_type=form' % (invoice._name, invoice.id, invoice.env.ref('account.action_move_out_invoice_type').id)"/>
+                    <t t-set="backend_url" t-value="'/odoo/action-%s/%s' % (invoice.env.ref('account.action_move_out_invoice_type').id, invoice.id)"/>
```
Comment (h4818) [inline addons/account/views/account_portal_templates.xml:107]: I only just learnt this from the diff below...
Could it be
```suggestion
                    <t t-set="backend_url" t-value="'/odoo/action-account.action_move_out_invoice_type/%s' % invoice.id)"/>
```

**Review by h4818 — APPROVED** [review]
LGTM, just a question

also, maybe rebase for runbot?

File: addons/account/views/account_portal_templates.xml (hunk 1 above)
Comment (sauc-odoo) [reply addons/account/views/account_portal_templates.xml]: Hello @h4818 and @jpp-odoo 

Changes have been made, please check it.
Thanks :slightly_smiling_face:

**Comment by jpp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**